crates/mun/src/main.rs

```diff
@@ -136,6 +142,29 @@ fn compiler_options(matches: &ArgMatches) -> Result<mun_compiler::CompilerOption
         _ => return Err(format_err!("Only optimization levels 0-3 are supported")),
     };
 
+    let color = match matches.value_of("color") {
```

Comment:
@baszalmstra  `value_of("color")` returns `&str`, but `env::var("MUN_TERM_COLOR")` returns `String`. I can't transform output from `env::var("MUN_TERM_COLOR")` to `&str` because of `returns a value referencing data owned by the current function`. Here's my current _not_ working code:
```
 let color = matches
        .value_of("color")
        .or_else(|| {
            env::var("MUN_TERMINAL_COLOR")
                .ok()
                .map(|value| value.as_str())
        })
        .map(|value| match value {
            "disable" => Color::Disable,
            "enable" => Color::Enable,
            _ => Color::Auto,
        })
        .unwrap_or(Color::Auto);
```
Any ideas?

Reply:
from the top of my head, first convert the `&str` to an owned string with `String::to_owned` :
```
matches
  .value_of("color")
  .map(ToOwned::to_owned)
  .or_else(|| env::var("MUN_TERM_COLOR")
  .map(|str| match &str { ... })
  .unwrap_or(Color::Auto)
```